indico/modules/events/papers/templates/_contributions.html

```diff
@@ -31,6 +36,56 @@ <h3 class="icon-file">{% trans %}Paper{% endtrans %}</h3>
     {% endif %}
 {% endmacro %}
 
+{% macro render_editables_section(contribution) %}
+    {% set editable = contribution.editables[0] if contribution.editables if contribution.editables else None %}
+    {% set show_editable = true if editable else false %}
+    {% set can_submit = contribution.is_user_associated(session.user, check_abstract=true) %}
+
+    {% if can_submit %}
+        <section>
+            <div class="header">
+                <div class="header-row">
+                <h2>{% trans %}Editing{% endtrans %}</h2>
+                </div>
+            </div>
+            <div class="header">
+                <div class="header-row">
+                    <h3 class="icon-file">{% trans %}Paper{% endtrans %}</h3>
+                </div>
+            </div>
+            {% if show_editable %}
+                <div>
+                    <a href="{{ url_for('event_editing.editable', contribution.event, contrib_id=contribution.id, type='paper') }}">
```

Comment:
you can just pass the `editable` to `url_for` instead of the kwargs since there are locators now that provide all the necessary data